src/style.css

```diff
@@ -1 +1,87 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+:root {
+  --text-font: Avenir, sans-serif;
+}
+
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.body {
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 20px;
+}
+
+.form__searchBar {
+  box-sizing: border-box;
+  font-family: var(--text-font);
+  font-size: 16px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 48%;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  color: #3d4e61;
+  outline: none;
+}
+
+.form__searchBar::placeholder {
+  font-weight: normal;
+  color: inherit;
+}
+
+.form__searchBar:hover {
+  box-shadow: 1px 4px 4px 1px rgba(0, 0, 0, 0.25);
+}
+
+.form__searchBar:focus {
+  font-weight: bold;
+  border-radius: 3px;
+  background: url(images/Search.svg),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  background-position:
+    25px 48%,
+    center;
+  box-shadow: 1px 4px 4px 1px #00000040;
+}
+
+.form__searchBar--big {
+  height: 70px;
+  font-weight: 16px;
```

Comment:
The `.form__searchBar--big` class has a `font-weight` property with a value of `16px`, which is incorrect. The `font-weight` property should be a keyword or number without a unit, such as `bold` or `400`.